src/config.ts

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
+// config settings, env variables
+import * as dotenv from 'dotenv';
+import * as path from 'path';
+
+import { type RedisConfig } from '@frmscoe/frms-coe-lib/lib/interfaces';
+
+// Load .env file into process.env if it exists. This is convenient for running locally.
+dotenv.config({
+  path: path.resolve(__dirname, '../.env'),
+});
+
+export interface IConfig {
+  maxCPU: number;
+  env: string;
+  functionName: string;
+  ruleName: string;
+  ruleVersion: string;
+  apm: {
+    secretToken: string;
+    serviceName: string;
+    url: string;
+    active: boolean;
+  };
+  db: {
+    name: string;
+    password: string;
+    url: string;
+    user: string;
+    dbCertPath: string;
+    cacheEnabled?: boolean;
+    cacheTTL?: number;
+  };
+  cmsProducer: string;
+  logger: {
+    logstashHost: string;
+    logstashPort: number;
+    logstashLevel: string;
+  };
+  redis: RedisConfig;
+  sidecarHost: string;
+  suppressAlerts: boolean;
+}
+
+export const config: IConfig = {
+  maxCPU: parseInt(process.env.MAX_CPU!, 10) || 1,
+  ruleName: process.env.RULE_NAME!,
+  ruleVersion: process.env.RULE_VERSION!,
+  apm: {
+    serviceName: process.env.APM_SERVICE_NAME!,
+    url: process.env.APM_URL!,
+    secretToken: process.env.APM_SECRET_TOKEN!,
+    active: process.env.APM_ACTIVE === 'true',
+  },
+  cmsProducer: process.env.CMS_PRODUCER!,
```

Comment:
Would prefer `interdictionProducer` / `INTERDICTION_PRODUCER` here to differentiate from the CMS push out of the TADProc. The EFRuP itself does not send traffic to the CMS endpoint.